src/BlueSCSI.cpp

```diff
@@ -1230,25 +1230,33 @@ void loop()
   SCSI_OUT(vCD ,  active) // gpio_write(CD, high);
   SCSI_OUT(vIO ,inactive) // gpio_write(IO, low);
   
-  int len;
   byte cmd[12];
-  cmd[0] = readHandshake(); if(m_isBusReset) goto BusFree;
+  int cmdCount;
+  cmd[0] = readHandshake(); 
+  if(m_isBusReset) 
+    goto BusFree;
+
   LOGHEX(cmd[0]);
-  // Command length selection, reception
-  static const int cmd_class_len[8]={6,10,10,6,6,12,6,6};
```

Comment:
Looks like one command has 12 - or it could be a typo/bug.

Reply:
I'm not sure, this code was based on what I found in the raSCSI code base.
https://github.com/akuker/RASCSI/blob/master/src/raspberrypi/gpiobus.cpp "GPIOBUS::CommandHandShake"
with this change it was more stable on the Amiga (not 100% but better then with out it).
Its possible that BlueSCSI is implementing a command that raSCSI doesn't or that there is a mistake in blue or in ra.
its hard to tell.

Reply:
Looking up some documention it seems both might be wrong in some way.
as there is a 16byte command as well.
though looking at the code none of the implemented commands seem to use more then the first 8 bytes.
![image](https://user-images.githubusercontent.com/18196946/117578448-4b772c00-b0e6-11eb-948c-f03eeff2044c.png)
https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf

Reply:
Its actually what I've done in an experimental branch and it seemed fine.
I've been checking out a bunch of other STM32 based scsi source codes and most only implement 3 group codes.
so this having all of them is overkill, but doesn't harm the code.